mmdet/core/fp16/hooks.py

```diff
@@ -97,8 +97,12 @@ def wrap_fp16_model(model):
 def patch_norm_fp32(module):
     if isinstance(module, (nn.modules.batchnorm._BatchNorm, nn.GroupNorm)):
         module.float()
-        module.forward = patch_forward_method(module.forward, torch.half,
-                                              torch.float)
+        path_forward_condition = isinstance(module, nn.GroupNorm)
+        major, minor = [int(v) for v in torch.__version__.split('.')][:2]
+        path_forward_condition |= major < 1 or major == 1 and minor < 3
+        if path_forward_condition:
```

Comment:
The condition can be rewritten in just one line:
```python
if torch.__version__ < (1, 3) or isinstance(module, nn.GroupNorm):
    module.forward = patch_forward_method(module.forward, torch.half, torch.float)
```

Reply:
![image](https://user-images.githubusercontent.com/16317197/75418899-a7242200-5945-11ea-81dc-3a83c0e9961e.png)
but I can write
![image](https://user-images.githubusercontent.com/16317197/75418952-c4f18700-5945-11ea-857b-b4abe590ff56.png)
It looks more pretty, thanks